.run/Reference Server_ default.run.xml

```diff
@@ -1,9 +1,7 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Reference Server: default" type="JetRunConfigurationType" folderName="Run Single Server">
     <envs>
-      <env name="app.enableTest" value="true" />
```

Comment:
Why is it not required anymore?

Reply:
The `enableTest` is already defined in [reference-config.yaml ](https://github.com/lijamie98/java-stellar-anchor-sdk/blob/feature/anchor-868-servicerunner-run-test-profile/service-runner/src/main/resources/config/reference-config.yaml#L17)